x-pack/plugins/aiops/public/embeddable/embeddable_chart_component_wrapper.tsx

```diff
@@ -235,16 +244,27 @@ export const ChartGridEmbeddableWrapper: FC<
       `}
     >
       {changePoints.length > 0 ? (
-        <ChartsGrid
-          changePoints={changePoints.map((r) => ({ ...r, ...fieldConfig }))}
-          interval={requestParams.interval}
-          onRenderComplete={onRenderComplete}
-        />
-      ) : emptyState ? (
-        emptyState
-      ) : (
-        <NoChangePointsWarning onRenderComplete={onRenderComplete} />
-      )}
+        visType === EMBEDDABLE_CHANGE_POINT_CHART_TYPE ? (
+          <ChartsGrid
+            changePoints={changePoints.map((r) => ({ ...r, ...fieldConfig }))}
+            interval={requestParams.interval}
+            onRenderComplete={onRenderComplete}
+          />
+        ) : visType === EMBEDDABLE_CHANGE_POINT_TABLE_TYPE ? (
+          <ChangePointsTable
```

Comment:
For the table embeddable, why don't we have the 'Edit' option in the dashboard panel menu, for example if I want to change the max number of rows?

<img width="239" alt="image" src="https://github.com/elastic/kibana/assets/7405507/e1c91f61-683f-4db8-8177-cfad6f3e83da">

Reply:
Edit action works as expected now